src/Statistics.h

```diff
@@ -0,0 +1,64 @@
+//Copyright (c) 2019 Ultimaker B.V.
+
+
+#ifndef STATISTICS_H
+#define STATISTICS_H
+
+#include "utils/polygon.h"
+#include "utils/ExtrusionJunction.h"
+#include "utils/ExtrusionSegment.h"
+#include "utils/ExtrusionLine.h"
+#include "utils/types/Duration.h"
+#include "SkeletalTrapezoidation.h"
+
+namespace arachne
+{
+
+/*!
+ * Get statistics of the resulting toolpaths
```

Comment:
This is a very, very short description of what the class does. This could use some expansion.